### PR TITLE
PostMinePlayback preference

### PR DESCRIPTION
**Pull request by artjomsR**
As discussed, 3 options to choose from for the new setting

### CODE REVIEW

File: common/settings/settings.ts [hunk 1]
```diff
@@ -39,6 +39,7 @@ export interface AnkiSettings {
     readonly maxImageHeight: number;
     readonly surroundingSubtitlesCountRadius: number;
     readonly surroundingSubtitlesTimeRadius: number;
+    readonly clickToMineDefaultPlayback: PostMinePlayback;
```
Comment (killergerbah) [inline common/settings/settings.ts]: Looking at this now the settings categorization does seem somewhat arbitrary but I think I would still prefer for this setting to be under `MiscSettings`.

Comment (killergerbah) [inline common/settings/settings.ts]: I would name this setting something else since it applies even when not clicking to mine.

**Review by killergerbah — CHANGES_REQUESTED** [review]

**Review by killergerbah — APPROVED** [review]

**Comment by killergerbah** [comment]
Thanks as always @artjomsR

**Comment by artjomsR** [comment]
Always a pleasure, thanks for your hard work on this beautiful project!

**Pull request merged**